fileutil/load.go

```diff
@@ -0,0 +1,39 @@
+package fileutil
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/juju/errors"
+)
+
+const (
+	// FileSource specifies to load config from a file
+	FileSource = "file://"
+	// EnvSource specifies to load config from an environment variable
+	EnvSource = "env://"
+)
+
+// LoadConfigWithSchema returns a configuration loaded from file:// or env://
+// If config does not start with file:// or env://, then the value is returned as is
+func LoadConfigWithSchema(config string) (string, error) {
+	if strings.HasPrefix(config, FileSource) {
+		fn := strings.TrimPrefix(config, FileSource)
+		f, err := ioutil.ReadFile(fn)
+		if err != nil {
+			return config, errors.Trace(err)
+		}
+		// file content
+		config = string(f)
```

Comment:
@dissoupov  Do you care if config == "" in this case? I see you check it for ENV case.